README.md

```diff
@@ -42,7 +42,7 @@ When creating a new `RegisterClientManager`, you can pass a configuration object
 
 Gets the `RegisterClient` instance for the given `register` name and `phase`.
 
-The `data_store` parameter specifies the data store to use accessing a particular register. By default this parameter can be omitted to use the `InMemoryDataStore`; however, a custom data store can be created (for example, to insert register data directly into your Postgres database) which `includes DataStore` module and implements the methods it defines.
+The `data_store` parameter specifies the data store to use accessing a particular register. You can omit this parameter, which will make it default to the `InMemoryDataStore` value. You can also create a custom data store to include the `DataStore` module and to implement the methods it defines. For example, to insert register data directly into your Postgres database. 
```

Comment:
Two points:
1. Potentially remove the word **value** after _which will make it default to the `InMemoryDataStore` value_
2. I think that it might make more sense to talking about _by including_ the `DataStore` module, rather than _to include_ the module. _You can also create a custom data store by including the `DataStore` module and implementing methods it defines. For example, to insert register data directly into your Postgres database._

Reply:
No worries about point 1, though it did occur to me that the `InMemoryDataStore` value isn't explicitly referenced anywhere else but here, so I wasn't sure what context (if any) to provide.

re. point 2, is that because the custom data store will do more than just include the `DataStore` module? If so I can certainly see how _including_ would make more sense.